numba/core/runtime/_nrt_python.c

```diff
@@ -286,7 +286,7 @@ PyObject* try_to_return_parent(arystruct_t *arystruct, int ndim,
 }
 
 NUMBA_EXPORT_FUNC(PyObject *)
-NRT_adapt_ndarray_to_python(arystruct_t* arystruct, int ndim,
+NRT_adapt_ndarray_to_python_acqref(arystruct_t* arystruct, int ndim,
```

Comment:
Whilst highly unlikely, any chance anyone else is using this? I did a quick scan of github and nothing jumped out.

Reply:
It has to be very unusual for anyone to be using this. Also external code are advised to use: https://numba.readthedocs.io/en/stable/developer/numba-runtime.html?highlight=runtime#using-the-nrt-from-c-code which does not expose this function.

Reply:
Agree about it being unusual, I think given docs do not expose this it can be considered internal only and changed.